File: go/1165.go
package main

import "fmt"

func isPrime(n int) bool {
	for i := 2; i < n; i++ {
		if n%i == 0 {
			return false
		}
	}

	return true
}

func main() {
	var n, x int

	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d", &x)

		if isPrime(x) {
			fmt.Printf("%d eh primo\n", x)
			continue
		}

		fmt.Printf("%d nao eh primo\n", x)
	}
}

File: lua/1165.lua
local function isPrime(n)
    if n <= 1 then
        return false
    end

    for i = 2, math.floor(math.sqrt(n)) do
        if n % i == 0 then
            return false
        end
    end

    return true
end

local n = io.read("*n")

for i = 1, n do
    local x = io.read("*n")

    if isPrime(x) then
        print(x .. " eh primo")
    else
        print(x .. " nao eh primo")
    end
end
